modules/nw-egress-ips-object.adoc

```diff
@@ -47,11 +47,11 @@ The following YAML describes the optional stanza for the pod selector:
 ----
 podSelector: <1>
   matchLabels:
-    name: <pod_name>
+    <label_name>: <label_value>
 ----
 <1> Optional: One or more matching rules for pods in the namespaces that match the specified `namespaceSelector` rules. If specified, only pods that match are selected. Others pods in the namespace are not selected.
 
-In the following example, the `EgressIP` object associates the `192.168.126.11` and `192.168.126.102` egress IP addresses with the pod that is named `my-pod` in the namespace that is named `my-namespace`:
+In the following example, the `EgressIP` object associates the `192.168.126.11` and `192.168.126.102` egress IP addresses with pods that have the `app` label set to 'web' and in the namespaces that have the label `env` set to `prod`:
```

Comment:
* 'web' -> `web`
* "and in" -> "and are in" ?
* "have the label `env` set" -> "have the `env` label set"